src/Drupal/Commands/core/DrupalCommands.php

```diff
@@ -57,39 +42,32 @@ public function __construct(CronInterface $cron, ModuleHandlerInterface $moduleH
 
     /**
      * Run all cron hooks in all active modules for specified site.
-     *
-     * @command core:cron
-     * @aliases cron,core-cron
-     * @usage drush maint:status && drush core:cron
-     *  Run cron unless maintenance mode is enabled
-     * @topics docs:cron
      */
+    #[CLI\Command(name: 'core:cron', aliases: ['cron', 'core-cron'])]
+    #[CLI\Usage(name: 'drush maint:status && drush core:cron', description: 'Run cron unless maintenance mode is enabled')]
+    #[CLI\Topics(topics: ['docs:cron'])]
     public function cron(): void
     {
         $this->getCron()->run();
     }
 
     /**
      * Information about things that may be wrong in your Drupal installation.
-     *
-     * @command core:requirements
-     * @option severity Only show status report messages with a severity greater than or equal to the specified value.
-     * @option ignore Comma-separated list of requirements to remove from output. Run with --format=yaml to see key values to use.
-     * @aliases status-report,rq,core-requirements
-     * @usage drush core:requirements
-     *   Show all status lines from the Status Report admin page.
-     * @usage drush core:requirements --severity=2
-     *   Show only the red lines from the Status Report admin page.
-     * @table-style default
-     * @field-labels
-     *   title: Title
-     *   severity: Severity
-     *   sid: SID
-     *   description: Description
-     *   value: Summary
-     * @default-fields title,severity,value
-     * @filter-default-field severity
      */
+    #[CLI\Command(name: 'core:requirements', aliases: ['status-report', 'rq', 'core-requirements'])]
+    #[CLI\Option(name: 'severity', description: 'Only show status report messages with a severity greater than or equal to the specified value.')]
+    #[CLI\Option(name: 'ignore', description: 'Comma-separated list of requirements to remove from output. Run with --format=yaml to see key values to use.')]
+    #[CLI\Usage(name: 'drush core:requirements', description: 'Show all status lines from the Status Report admin page.')]
+    #[CLI\Usage(name: 'drush core:requirements --severity=2', description: 'Show only the red lines from the Status Report admin page.')]
+    #[CLI\FieldLabels(labels: [
+        'title' => 'Title',
+        'severity' => 'Severity',
+        'sid' => 'SID',
+        'description' => 'Description',
+        'value' => 'Summary',
+    ])]
+    #[CLI\DefaultFields(fields: ['severity', 'value'])]
```

Comment:
Was there a reason you took out `title` from the default fields? Another note: in most cases, it's probably better to use `DefaultTableFields` instead of `DefaultFields` now. The former applies only to "human readable" formats, whereas the later applies to all formats. If DefaultFields is omitted (recommended), then formats such as json will default to showing all available information.

Reply:
Fixed. Now using DefaultTableFields everywhere.